pdf/core/parser.go

```diff
@@ -1498,6 +1503,7 @@ func NewParserFromString(txt string) *PdfParser {
 	parser.reader = bufferedReader
 
 	parser.fileSize = int64(len(txt))
+	parser.streamLengthReferenceLookupInProgress = map[int64]bool{}
```

Comment:
Any reason to initialize here?  Fixes a bug or something?

Reply:
This is needed for font_test.go `parsePdfObjects`.